File: Bimestre 2/funciones/FuncionesYProc/src/funcionesyproc/fun5_2B.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package funcionesyproc;
import java.util.Scanner;
/**
 *
 * @author kevin
 */
public class fun5_2B {
    public static void main(String[] args) {
        Scanner leer = new Scanner(System.in);
        int i, potP,numP, denP;
        double suma,factP,n;
        suma = 0;numP= -1; denP=0; potP=0; factP=0;
        System.out.print("Ingrese el limite de la serie: ");
        n = leer.nextInt();
        
        for (i = 1; i <=n ; i++) {
            numP = Numerador(numP);
            denP = Denominador(denP);
            factP = Factorial(denP);
            potP = Potencia(potP);
            
            System.out.println(numP + "^" + potP + "/"+denP+"!");
        
            suma = suma + ((numP^potP)/factP);
            
        }
        System.out.println("La suma de la serie númerica es: "+suma);
    }
    
    public static int Numerador(int num){
        num = num + 2;
        return num;
    }
    public static int Denominador( int den){
        den = den + 2;
        return den;
    }
    public static int Factorial( double den){
        int fact, j;
        fact = 1;
        for (j = 1; j <= den; j++) {
            fact = fact * j;
        }
        return fact;
    }
    public static int Potencia(int pot){
        pot = pot + 3;
        return pot;
        
    }
}

File: Bimestre 2/funciones/funcionesyProPython/fun5_2B.py
def Numerador(num):
    num = num+2
    return num
def Denominador(den):
    den = den + 2
    return den
def Factorial(den):
    fact=1
    for j in range(1,den+1):
        fact = fact * j
    return fact
def Potencia(pot):
    pot = pot + 3
    return pot
numP= -1
denP=0
potP=0
factP=0
suma=0
n = int(input('Ingrese el limite de la serie: '))
for i in range(1,n+1):
    numP = Numerador(numP)
    denP = Denominador(denP)
    factP = Factorial(denP)
    potP = Potencia(potP)
    print(str(numP) + "^" + str(potP) + "/"+ str(denP) +"!")

    suma = suma + ((numP^potP)/factP)
print("La suma de la serie númerica es: "+ str(suma))